setup.py

```diff
@@ -48,8 +48,7 @@
 # though that is installed out of band (managed by the extension)
 # for compatibility reasons.
 
-DEPENDENCIES = ["paho-mqtt==1.5.0", "jsonschema==3.2.0", "packaging"]
-
+DEPENDENCIES = ["paho-mqtt>=1.5.0", "jsonschema==3.0.2", "packaging", "azure-iot-device>=2.5.1"]
```

Comment:
if we add `azure-iot-device` here we should remove it from dev_requirements

Reply:
IIRC `urllib3[secure]>=1.21.1,<=1.25` in dev_requirements was specifically for `azure-iot-device` so it should be removed from there as well.